message-index/messages/GHC-11861/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Empty single quotes 
+summary: No character literal provided within single quotes
+severity: error
+introduced: 9.6.1
+---
+
+Single quotes are used for character and literals including "new line" (\n), "carriage return" (\r), "horizontal tab" (\t), and "vertical tab" (\v). Check list of literals in [documentation](https://www.haskell.org/onlinereport/lexemes.html): 2.6  Character and String Literals. Character literals cannot be empty.
+
+Another usage of single quotes is in TemplateHaskell, please check [documentation](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/template_haskell.html#syntax).
```

Comment:
I think we should link to released version documentation here (https://downloads.haskell.org/ghc/latest/docs/html/users_guide/) because most people aren't using development versions of GHC.

Reply:
kk, updated